recipes/azure-core-cpp/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set version = "1.10.0" %}
+{% set sha256 = "e6ae26ad30376bef6272024564764ba7bf5756063dc94cc4b74180cd004ab75e" %}
+
+package:
+  name: azure-core-cpp
+  version: {{ version }}
+
+source:
+  url: https://github.com/Azure/azure-sdk-for-cpp/archive/refs/tags/azure-core_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # https://github.com/Azure/azure-sdk-for-cpp/issues/4784
+    # https://github.com/Azure/azure-sdk-for-cpp/pull/4785
+    - 0001-remove-wil-from-exported-config.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("azure-core-cpp", min_pin="x", max_pin="x") }}
+
+# https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core/vcpkg.json
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake
+    - ninja
+  host:
+    - libcurl
+    - openssl  # [not win]
+    - wil  # [win]
+
+test:
+  commands:
+    # shared libs
+    - test -f ${PREFIX}/lib/libazure-core.so            # [linux]
+    - test -f ${PREFIX}/lib/libazure-core.dylib         # [osx]
+    - if not exist %LIBRARY_BIN%\azure-core.dll exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\azure-core.lib exit 1  # [win]
+
+    # headers
+    - test -f ${PREFIX}/include/azure/core.hpp    # [unix]
+    - if not exist %LIBRARY_INC%\azure\core.hpp exit 1  # [win]
+
+    # CMake metadata
+    - test -f ${PREFIX}/share/azure-core-cpp/azure-core-cppTargets.cmake                     # [unix]
+    - if not exist %LIBRARY_PREFIX%\share\azure-core-cpp\azure-core-cppTargets.cmake exit 1  # [win]
+
+about:
+  home: https://github.com/Azure/azure-sdk-for-cpp/
+  summary: Azure SDK Core Library for C++
+  description: >-
+    Azure::Core (`azure-core`) provides shared primitives, abstractions, and
+    helpers for modern Azure SDK client libraries written in the C++
+  license: MIT
+  license_family: MIT
+  license_file:
+    - LICENSE.txt
+    - sdk/core/azure-core/NOTICE.txt
+  doc_url: https://azure.github.io/azure-sdk-for-cpp/
+  dev_url: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/core/azure-core/
+
+extra:
+  recipe-maintainers:
+    - jdblischak
+    - Shelnutt2
```

If you need another maintainer.
```suggestion
    - Shelnutt2
    - jjerphan
```

---

I'd love another maintainer. Thanks for volunteering!